### PR TITLE
Fix problem with display of structure levels

### PR DESCRIPTION
**Pull request by reiterl**
In default times = 0 display all structure levels of a user

### CODE REVIEW

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts [hunk 1]
```diff
@@ -91,6 +91,10 @@ export class ListOfSpeakersContentComponent extends BaseMeetingComponent impleme
         return this.meetingSettingService.get(`list_of_speakers_default_structure_level_time`).pipe(map(v => v > 0));
     }
 
+    public get structureLevelCountdownDisabled(): Observable<boolean> {
+        return this.meetingSettingService.get(`list_of_speakers_default_structure_level_time`).pipe(map(v => v == 0));
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts:95]: Always use exact checks if possible. 

```suggestion
        return this.meetingSettingService.get(`list_of_speakers_default_structure_level_time`).pipe(map(v => v === 0));
```

**Review by bastianjoel — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts (hunk 1 above)
Comment (reiterl) [reply client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts]: Done.

**Review by bastianjoel — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]
function

**Comment by bastianjoel** [comment]
This is not compatible with the changes requested in https://github.com/OpenSlides/openslides-client/issues/3166

**Pull request merged**